nixos/lib/make-ext4-fs.nix

```diff
@@ -37,32 +37,28 @@ pkgs.stdenv.mkDerivation {
       ${populateImageCommands}
       )
 
-      # Add the closures of the top-level store objects.
-      storePaths=$(cat ${sdClosureInfo}/store-paths)
+      # Create nix/store before copying path
+      mkdir -p ./rootImage/nix/store
+
+      # Also include a manifest of the closures in a format suitable for nix-store --load-db
+      cp ${sdClosureInfo}/registration ./rootImage/nix-path-registration
+
+      echo "copying store paths for image..."
+      cp -a $(cat ${sdClosureInfo}/store-paths) -t ./rootImage/nix/store/
+
+      echo "copying files for image..."
```

Comment:
What "files"?

Reply:
I have no idea, this is copypaste from old version. Should i remove this line? It's not really informative.

Reply:
Yeah, I think it's better; the message is more noise than signal :P

Reply:
Removed.